clients/src/main/java/org/apache/kafka/common/security/ssl/SslFactory.java

```diff
@@ -222,4 +241,99 @@ private KeyStore load() throws GeneralSecurityException, IOException {
         }
     }
 
+
+    private class ReloadableX509TrustManager extends X509ExtendedTrustManager implements X509TrustManager {
+        private final SecurityStore trustStore;
+        private TrustManagerFactory tmf;
+        private X509TrustManager trustManager;
+        private long lastReload = 0L;
+        private static final long MINIMAL_WAIT = 60 * 1000;
```

Comment:
Not sure if the reload interval needs to be configurable, but at least it needs to be a variable to enable testing.